src/main/java/com/github/nkzawa/engineio/client/Socket.java

```diff
@@ -305,6 +305,8 @@ private void probe(final String name) {
         final Transport[] transport = new Transport[] {this.createTransport(name)};
         final boolean[] failed = new boolean[] {false};
         final Socket self = this;
+        
+        self.emit(EVENT_TRANSPORT, transport);
```

Comment:
Why not? I'm not a Java expert...
I've given you access to: https://github.com/adrai/engine.io-client.java so if you want to change it or close this PR... up to you, ok?